SPECS/kernel/config_aarch64

```diff
@@ -2040,6 +2040,7 @@ CONFIG_PCI_IOV=y
 CONFIG_PCI_PRI=y
 CONFIG_PCI_PASID=y
 CONFIG_PCI_LABEL=y
+# CONFIG_PCI_HYPERV is not set
```

Comment:
I think you'll want to set these PCI_HYPERV configs

Reply:
Turned on the configs for pci-hyperv